### DISCUSSION TITLE
authcrypt option in didcomm

### DISCUSSION THREAD

**Discussion by ismapolis** [decentralized-identity/veramo#1171] [Q&A] [answered]
Hello Ijust would like to know why "authcrypt" option is omitted in the switch.

![image](https://user-images.githubusercontent.com/53060433/235309659-1f4c47e1-f8ac-4578-bd8b-bfcd7b97b836.png)

**Comment by mirceanis** [comment, marked as answer]
Hi, it's not omitting `'authcrypt'`. It's omitting the `break` between the switch cases.
This means that for both `authcrypt` and `anoncrypt` the method `this.packDIDCommMessageJWE` will be called.

BTW, it's easier to get answers if you paste [links to code](https://github.com/uport-project/veramo/blob/c965fd502f652c9929ae4753c56ebbe351447733/packages/did-comm/src/didcomm.ts#L163) instead of screenshots. It's a big codebase so sometimes it's hard to figure out where the screenshot is coming from.
